repotracker/repotracker.go

```diff
@@ -621,11 +621,32 @@ func CreateVersionFromConfig(ctx context.Context, projectInfo *model.ProjectInfo
 	}
 	v.Ignored = ignore
 
+	// Compute aliases first, so we can include this in the version errors if there's a problem.
```

Comment:
Could this comment say something specific about what we are including in the version errors so future readers don't think the below segment should be adding to project errors rather than returning an error (L630 & L635)?

Maybe like `Compute aliases first to include undefined requested alias in version errors."